### DISCUSSION TITLE
Modifying Foot IK Offset

### DISCUSSION THREAD

**Discussion by sommerlost** [Sixze/ALS-Refactored#89] [Q&A] [answered]
Hi there, I'm working on learning more about the refactored version of ALs. I've been digging around in the Animation Blueprint, but can't seem to find the animation layers for Foot IK. In the past I was able to add a simple offset to the pelvis and each foot to deal with characters of different heights and make sure that the feed are touching the ground.  Is this still exposed to Blueprint? Thanks!

![image](https://user-images.githubusercontent.com/7546149/173201269-f921db75-b803-4854-9056-b5519edc4d01.png)

![image](https://user-images.githubusercontent.com/7546149/173201260-e5d22268-8e56-4f80-a623-24f7a8fa25c4.png)

-----------

After more inspection, I thought I could modify this in the Control Rig (CR_Als), but it doesn't seem to have much effect of the character.

**Comment by Sixze** [comment, marked as answer]
Take a look at `CR_Als`, all bone manipulation work is now done here.

**Reply by sommerlost** [reply]
Brilliant. I love the use of control rig over the old system. I just don't think I was digging deep enough into the Control Rig yesterday. For anyone looking, I added a pelvis offset here:

![image](https://user-images.githubusercontent.com/7546149/173244808-7d61c44b-0c4e-456e-a2cb-b989c902bbd8.png)
